pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
+
+const (
+	Nano  Scale = -9
+	Micro Scale = -6
+	Milli Scale = -3
```

Comment:
not clear why this list doesn't include Kilo, Kibi, Mega, Mebi, etc.  I presume it's because these are only the decimal-less-than-0 values - comment would help